spinnaker/spinnaker_system/google_front50_test.py

```diff
@@ -97,12 +102,13 @@ def create_app(self):
     # file.
     gcs_builder = gcp.GoogleCloudStorageContractBuilder(self.gcs_observer)
     (gcs_builder.new_clause_builder('Created Google Cloud Storage File')
-     .list(self.BUCKET_ROOT, 'applications')
+     .list(self.BUCKET, os.path.join(self.BASE_PATH, 'applications'))
```

Comment:
`os.path.join` will join the args based on the path separator that the current os thinks is the path separator. This is probably fine for our use cases, but on Windows this might break.